### PR TITLE
AK: Add to_radians and to_degrees math functions

### PR DESCRIPTION
**Pull request by bplaat**
This adds the `to_radians` and `to_degrees` functions to AK math, I have also updated all instances in the codebase where degrees where converted to radians and reverse, maybe I missed some then they can also be added to this commit.

These functions make the code a little bit easier to read.

### CODE REVIEW

File: Userland/Libraries/LibWeb/SVG/SVGGraphicsElement.cpp [hunk 1]
```diff
@@ -75,7 +75,7 @@ Gfx::AffineTransform transform_from_transform_list(ReadonlySpan<Transform> trans
 {
     Gfx::AffineTransform affine_transform;
     auto to_radians = [](float degrees) {
-        return degrees * (AK::Pi<float> / 180.0f);
+        return AK::to_radians(degrees);
     };
```
Comment (MacDue) [inline Userland/Libraries/LibWeb/SVG/SVGGraphicsElement.cpp:79]: Looks like you can get rid of this little lambda now and make its users call `AK::to_radians` directly.

Comment (bplaat) [reply Userland/Libraries/LibWeb/SVG/SVGGraphicsElement.cpp:79]: Good point

**Review by MacDue — APPROVED** [review]
LGTM, seems nice

**Review by kalenikaliaksandr — APPROVED** [review]

**Comment by timschumi** [comment]
:bikeshed:, but should those functions maybe be named `degrees_to_radians` and `radians_to_degrees` respectively as long as we don't have distinct numerics for these types?

That said, I don't know how well combining an arbitrary `FloatingPoint` with `DistinctNumeric` would work (or if it even does at all).

**Comment by bplaat** [comment]
I like the current names more than a more explicit version. But that is more of a personal preference really.

**Comment by Hendiadyoin1** [comment]
> :bikeshed:, but should those functions maybe be named `degrees_to_radians` and `radians_to_degrees` respectively

Tbh I'd even prefer the numpy names with math stuff, so `deg2rad` and `rad2deg`,
Math functions already have old acronymic names, so why stop there, when we can make use of existing muscle memory

**Comment by AtkinsSJ** [comment]
> Tbh I'd even prefer the numpy names with math stuff, so `deg2rad` and `rad2deg`,

:thousandnostare:

**Pull request merged**